apps/src/componentLibrary/common/styles/primitiveColors.css

```
@@ -1,5 +1,8 @@
-/* DO NOT IMPORT THIS FILE DIRECTLY. Use colors.css instead,
- which convey semantic and utility colors and also defines themes. */
+/* This is a collection of Primitive Colors.
+ Unlike semantic colors (color.css) this variables will be equal to same color value in all themes.
+ Ideally, these colors should be used for components that are not theme dependent.
+ In the future it's possible that we will only need to use these colors in semantic colors (colors.css).
```

Tiny consistency nit:
```suggestion
 In the future it's possible that we will only need to use these colors in Semantic colors (colors.css).
```